manifests/volume.pp

```diff
@@ -55,6 +55,7 @@
   Optional[Integer] $stripe                   = undef,
   Optional[Integer] $replica                  = undef,
   Optional[Integer] $arbiter                  = undef,
+  Optional[Boolean] $force_binary              = false,
```

Comment:
same here
```suggestion
  Boolean $force_binary              = false,
```

Reply:
@bastelfreak 
Without Optional is also okay, should i update it at my repo?

Reply:
yes please. You can update your code on your own or merge the suggestion

Reply:
@bastelfreak optional parameter have been removed.